apps/dashboard/app/javascript/packs/config.js

```diff
@@ -0,0 +1,35 @@
+'use strict;'
+
+const CONFIG_ID = 'ood_config'
+
+function configData() {
+  return $(`#${CONFIG_ID}`).data();
+}
+
+function setNavbarColor() {
+  const cfgData = configData();
+  const styles = document.styleSheets[0];
+
+  styles.insertRule(navbar('light', cfgData['bgColor']), styles.lenth);
```

Comment:
`styles.lenth` looks like a typo. Should it be `length`?

Reply:
Yep, you're right! If it's undefined, I wonder where it's inserting?  I guess it doesn't matter either way.